DXClient.sln

```diff
@@ -18,168 +18,168 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DTAConfig", "DTAConfig\DTAC
-		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.MORelease|WindowsGL.ActiveCfg = MORelease|WindowsGL
-		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.MORelease|WindowsGL.Build.0 = MORelease|WindowsGL
 		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.MORelease|SharpDX.ActiveCfg = MORelease|SharpDX
 		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.MORelease|SharpDX.Build.0 = MORelease|SharpDX
+		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.MORelease|WindowsGL.ActiveCfg = MORelease|WindowsGL
+		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.MORelease|WindowsGL.Build.0 = MORelease|WindowsGL
 		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.MORelease|XNAFramework.ActiveCfg = MORelease|XNAFramework
 		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.MORelease|XNAFramework.Build.0 = MORelease|XNAFramework
-		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.TIRelease|WindowsGL.ActiveCfg = TIRelease|WindowsGL
-		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.TIRelease|WindowsGL.Build.0 = TIRelease|WindowsGL
 		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.TIRelease|SharpDX.ActiveCfg = TIRelease|SharpDX
 		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.TIRelease|SharpDX.Build.0 = TIRelease|SharpDX
+		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.TIRelease|WindowsGL.ActiveCfg = TIRelease|WindowsGL
+		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.TIRelease|WindowsGL.Build.0 = TIRelease|WindowsGL
 		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.TIRelease|XNAFramework.ActiveCfg = TIRelease|XNAFramework
 		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.TIRelease|XNAFramework.Build.0 = TIRelease|XNAFramework
-		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.TSRelease|WindowsGL.ActiveCfg = TSRelease|WindowsGL
-		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.TSRelease|WindowsGL.Build.0 = TSRelease|WindowsGL
 		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.TSRelease|SharpDX.ActiveCfg = TSRelease|SharpDX
 		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.TSRelease|SharpDX.Build.0 = TSRelease|SharpDX
+		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.TSRelease|WindowsGL.ActiveCfg = TSRelease|WindowsGL
+		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.TSRelease|WindowsGL.Build.0 = TSRelease|WindowsGL
 		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.TSRelease|XNAFramework.ActiveCfg = TSRelease|XNAFramework
 		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.TSRelease|XNAFramework.Build.0 = TIRelease|XNAFramework
-		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.YRRelease|WindowsGL.ActiveCfg = YRRelease|WindowsGL
-		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.YRRelease|WindowsGL.Build.0 = YRRelease|WindowsGL
 		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.YRRelease|SharpDX.ActiveCfg = YRRelease|SharpDX
 		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.YRRelease|SharpDX.Build.0 = YRRelease|SharpDX
+		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.YRRelease|WindowsGL.ActiveCfg = YRRelease|WindowsGL
+		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.YRRelease|WindowsGL.Build.0 = YRRelease|WindowsGL
 		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.YRRelease|XNAFramework.ActiveCfg = YRRelease|XNAFramework
 		{D517317D-8EA8-4225-AF9E-5385D8A7F047}.YRRelease|XNAFramework.Build.0 = YRRelease|XNAFramework
```

Comment:
The reordering here is weird, but as far as I can see, no information / configuration choices have been actually lost.

Reply:
Most likely a consequence of not doing this entirely by hand and letting Visual Studio add the new configurations.